java/server/src/org/openqa/selenium/netty/server/NettyServer.java

```diff
@@ -46,19 +48,30 @@
   private final int port;
   private final URL externalUrl;
   private final HttpHandler handler;
+  private final SslContext sslCtx;
 
   private Channel channel;
 
-  public NettyServer(BaseServerOptions options, HttpHandler handler) {
+  public NettyServer(BaseServerOptions options, HttpHandler handler) 
+    throws SSLException, CertificateException {
```

Comment:
Consider wrapping the `SSLException` with an `UncheckedIOException` so that everything coming out of this is a `RuntimeException`. Our own code wraps and rethrows, so it's not like we're hiding much.